### PR TITLE
Bundle & package updates

### PR DESCRIPTION
**Pull request by jasoncypret**
So far I've just run `bundle update` and `npm update`

https://github.com/powerhome/nitro-storybook/network/dependencies

cc: @thatjsguy

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,9 +65,9 @@
     "node-sass": "4.5.3",
     "react": "15.3.2",
     "react-bootstrap": "0.32.1",
-    "react-color": "^2.13.8",
+    "react-color": "2.13.8",
```
Comment (thatjsguy) [inline package.json]: All deps should be static versions here

File: nitro_sg.gemspec [hunk 2]
```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config,db,lib}/**/*"] + ["Rakefile", "README.md"]
 
-  s.add_dependency "rails", ">= 4.2.9", "< 6.0"
+  s.add_dependency "rails", "~> 5.1.6"
```
Comment (thatjsguy) [inline nitro_sg.gemspec]: In `nitro-web`, we are exactly here `~> 5.1.6` for `rails`

File: README.md [hunk 3]
```diff
@@ -20,8 +20,8 @@ The intent of this repo is to provide a base on which other UIs can be built suc
 From the current project directory, run:
 
 1. ensure you are running proper node version (see `package.json` => `engines`)
-2. `npm install`
-3. `npm run storybook`
+2. `yarn install`
+3. `yarn run storybook`
```
Comment (thatjsguy) [inline README.md]: We should only be using `yarn`. Using `npm` will challenge our maintainability in ways that we cannot predict.

File: Gemfile.lock [hunk 4]
```diff
@@ -165,13 +168,13 @@ GEM
...
+    tzinfo (1.2.5)
       thread_safe (~> 0.1)
-    unicode-display_width (1.3.0)
-    websocket-driver (0.6.5)
+    unicode-display_width (1.4.0)
+    websocket-driver (0.7.0)
       websocket-extensions (>= 0.1.0)
     websocket-extensions (0.1.3)
-    yard (0.9.9)
+    yard (0.9.15)
```
Comment (thatjsguy) [inline Gemfile.lock]: One of the offending libs

File: Gemfile.lock [hunk 5]
```diff
@@ -1,129 +1,133 @@
...
       sprockets-rails (>= 2.0.0)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
-      nokogiri (>= 1.6)
-    rails-html-sanitizer (1.0.3)
-      loofah (~> 2.0)
-    railties (5.1.4)
-      actionpack (= 5.1.4)
-      activesupport (= 5.1.4)
+      nokogiri (>= 1.8.4)
```
Comment (thatjsguy) [inline Gemfile.lock]: One of the offending libs

File: Gemfile.lock [hunk 6]
```diff
@@ -1,129 +1,133 @@
...
+    erubi (1.7.1)
     ffi (1.9.25)
     globalid (0.4.1)
       activesupport (>= 4.2.0)
-    i18n (0.8.6)
+    i18n (1.0.1)
+      concurrent-ruby (~> 1.0)
     json (1.8.6)
-    loofah (2.1.1)
+    loofah (2.2.2)
```
Comment (thatjsguy) [inline Gemfile.lock]: One of the offending libs

File: Gemfile.lock [hunk 7]
```diff
@@ -1,129 +1,133 @@
...
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
-      nokogiri (>= 1.6)
-    rails-html-sanitizer (1.0.3)
-      loofah (~> 2.0)
-    railties (5.1.4)
-      actionpack (= 5.1.4)
-      activesupport (= 5.1.4)
+      nokogiri (>= 1.8.4)
+    rails-html-sanitizer (1.0.4)
```
Comment (thatjsguy) [inline Gemfile.lock]: One of the offending libs

File: Gemfile.lock [hunk 8]
```diff
@@ -1,129 +1,133 @@
 PATH
   remote: .
   specs:
-    nitro_sg (2.0.9)
-      rails (>= 4.2.9, < 6.0)
+    nitro_sg (2.2.2)
+      rails (~> 5.1.6)
```
Comment (thatjsguy) [inline Gemfile.lock]: Now using proper `rails` version

File: nitro_sg.gemspec (hunk 2 above)
Comment (benlangfeld) [reply nitro_sg.gemspec]: This makes it more difficult to upgrade nitro-web. What is the reason for this?

Comment (thatjsguy) [reply nitro_sg.gemspec]: Well we certainly shouldn't be on `4.x` whatsoever. How about `">= 5.1.6", "< 6.0"`? That should be flexible enough right?

**Pull request merged**